webxrlayers-1.bs

```diff
@@ -797,6 +800,69 @@ interface XRMediaBinding {
 
 </pre>
 
+WebXR Device API Integration {#webxrintegration}
+============================
+
+XRRenderState changes {#xrrenderstatechanges}
+---------------------
+This module extends the {{XRRenderStateInit}} and {{XRRenderState}} interfaces with a new optional array {{XRRenderStateInit/layers}} containing
+instances of {{XRLayer}} or {{XRWebGLLayer}}.
+
+<pre class="idl">
+typedef (XRWebGLLayer or XRLayer) XRGenericLayer;
+
+dictionary XRRenderStateInit {
+  double depthNear;
+  double depthFar;
+  double inlineVerticalFieldOfView;
+  XRWebGLLayer? baseLayer;
+  FrozenArray&lt;XRGenericLayer&gt;? layers;
+};
+
+[SecureContext, Exposed=Window] interface XRRenderState {
+  readonly attribute double depthNear;
+  readonly attribute double depthFar;
+  readonly attribute double? inlineVerticalFieldOfView;
+  readonly attribute XRWebGLLayer? baseLayer;
+  readonly attribute FrozenArray&lt;XRGenericLayer&gt;? layers;
+};
+</pre>
+
+The <dfn dict-member for="XRRenderStateInit">layers</dfn> is an array containing
+instances of {{XRLayer}} or {{XRWebGLLayer}} to hand to the [=XR Compositor=].
+
+The <dfn attribute for="XRRenderState">layers</dfn> attribute returns an array containing
+the instances of {{XRLayer}} or {{XRWebGLLayer}} that are displayed by the [=XR Compositor=].
+
+<div class="algorithm" data-algorithm="initialize-renderstate-for-layers">
+When an {{XRRenderState}} object |state| is created for an {{XRSession}} |session| with {{layers}} enabled, the user agent MUST <dfn dfn-for="layers">initialize the render state</dfn> by running the following steps:
+  1. Initialize |state| by running the steps to [=initialize the render state=].
```

Comment:
Is this self referential to the dfn in the previous line? I think the structure here is good, I just want to make sure the linking is working as intended.

Reply:
No, it refers to the definition in the WebXR spec. 
The one in the previous line has a `dfn-for="layers"` so it's different.

Reply:
It creates a link to the steps in WebXR.